CHANGELOG.md

```diff
@@ -25,6 +25,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.3.2] - 2022-09-22
```

Comment:
Per discussion with Jack, we're actually going to bump the major version here for two reasons:

* This is technically a backwards incompatible change if somebody was using cfs-config-util as a library in other Python code. Some classes have been removed for example.
* This gives us room for bugfixes on the release/3.3 branch if needed without pulling in this change too.